db/config.js

```diff
@@ -0,0 +1,19 @@
+const path = require('path');
+
+try {
+  require('dotenv').load({ path: path.resolve(__dirname, '..', '.env') });
+} catch (e) { }
+
+const config = {
+  username: process.env.PGUSER,
+  password: process.env.PGPASSWORD,
+  database: process.env.PGDATABASE,
+  host: process.env.PGHOST,
+  dialect: 'postgres',
+};
+
+module.exports = {
+  development: config,
+  test: config,
+  production: config,
```

Comment:
Yes and no. I think that I should probably just remove test and production keys there, hopefully, sequelize won't complain at us.

They're all the same here because we don't want to be storing any db credentials in the repo. We can run migrations on prod/staging by passing in custom connection strings as well.